index.html

```diff
@@ -10,8 +10,98 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <!-- Top Left Karma Logo -->
+    <header>
+        
+            <img src="img\karma-logo.svg" 
+            alt="Karma Logo" 
+            class="header__logo"/>
+        
+    <!-- Top Right Navigation Links-->
+    <nav>
+        <ul class="navigation__list">
+            <li class="navigation__item">
+                <a class="navigation__link" style="color:darkslategrey" href="#">Meet Karma</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">How It Works</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Store</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Blog</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Help</a>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Login</a>
+                </li>
+            </li>
+        </ul>
+    </nav>
+    </header>
+
+    <!-- Main -->
+    <main class="main_image"> 
+        <h1 class="main_intro">Introducing Karma</h1>
+        <h2 class="wifi_promo">
+            Bring WiFi with you, everywhere you go.
+        </h2>
+        <button class="learn_more_button">Learn More</button>
+    </main>
+
+    <!-- 1st Section  Heading -->
+    <h1 class="everyone_karma"> Everyone needs a little Karma</h1>
+
+    <!-- 1st Section Images -->
+        <section class="services_container">
+            <img  src="img\icon-devices.svg" alt="Internet Devices"/>
+            <img  src="img\icon-coffee.svg" alt=" Cup of Coffee"/>
+            <img  src="img\icon-refill.svg" alt="Refill"/>
+        </section >
+
+    <!-- 1st Section Icon Descriptions -->
+        <div class="icon_desc">
+            <h2>Internet for all devices</h2>
+            <h2>Boost your productivity</h2>
+            <h2>Pay as You Go</h2>
+        </div>
```

Comment:
Better to re-structure this so each heading 'lives' with its corresponding image. You can have a div to wrap each heading-image pair so you can easily use `display: flex` and `flex-direction: column` to stack them vertically.